README.md

```diff
@@ -62,15 +60,36 @@ twitter.stream("searchTerm", onTweet)
 An optional third parameter can be passed to the *Twitter.stream()*: onError. The onError parameter is a callback method that takes a single parameter (the error) and that will be invoked if any errors arise during the stream. If no callback is supplied, the default handler will be user:
 
 ```squirrel
-function onError(err) {
-  server.error(err);
+function onError(errors) {
```

Comment:
What is the format for the single error parameter?
(It looks like an array of tables with code/message keys in the example)